jazz_asset-event-handler/package.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Can you create a diff branch in your fork for this bug fix as this file doesn't belong to this PR?